seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/rest/RestHttpGetCommandProcessor.java

```diff
@@ -349,12 +355,31 @@ private void getRunningThread(HttpGetCommand command) {
                         .collect(JsonArray::new, JsonArray::add, JsonArray::add));
     }
 
-    private Map<String, Long> getJobMetrics(String jobMetrics) {
-        Map<String, Long> metricsMap = new HashMap<>();
+    private Map<String, Object> getJobMetrics(String jobMetrics) {
+        Map<String, Object> metricsMap = new HashMap<>();
         long sourceReadCount = 0L;
         long sinkWriteCount = 0L;
+        Map<String, JsonNode> tableSourceReceivedCountMap = new HashMap<>();
+        Map<String, JsonNode> tableSinkWriteCountMap = new HashMap<>();
         try {
             JsonNode jobMetricsStr = new ObjectMapper().readTree(jobMetrics);
+            StreamSupport.stream(
```

Comment:
please add test case to assert the metrics value use rest api in 
https://github.com/apache/seatunnel/blob/c645d925dcb85d6d0932427eba39a7e4283ec8ed/seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/java/org/apache/seatunnel/engine/e2e/ClusterSeaTunnelContainer.java#L447

Reply:
The MultiTableMetricsIT test case has been added